docs/examples/rasterio_backend_example.ipynb

```diff
@@ -20,24 +20,23 @@
   },
```

Comment:
<p>Could you add a sentence describing the overall purpose of this notebook? Something like "this notebook demonstrates how to use titiler-cmr to visualize data from one of the 2 HLS collections in CMR. It uses earthaccess to query for data, titiler-cmr to determine which assets to tile, and a combination of titiler-cmr and folium to render those assets in a map display."</p><p> </p><p>Perhaps a more descriptive tile as well, like "How to visualize HLS data in a common mapping client"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/titiler-cmr/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
The notebook is meant to show how you parameterize tilejson/tile requests for the `rasterio` backend, so I updated the description to reflect that. It's just a coincidence that it uses HLS!